.rubocop_todo.yml

```diff
@@ -53,6 +53,7 @@ Lint/MissingSuper:
     - "app/components/paginator/view.rb"
     - "app/components/providers/filters/view.rb"
     - "app/components/tab_navigation/view.rb"
+    - "app/components/allocation_record_card/view.rb"
```

Comment:
If this is here is it inheriting of the wrong parent 

Maybe use `ViewComponent::Base` or it really needs `super`